chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -54,6 +58,8 @@ listing.criteria.missingG2Related=(g)(2) was found without a required related ce
 listing.criteria.missingG1Related=(g)(1) was found without a required related certification.
 listing.criteria.missingComplementG3=A certification that requires (g)(3) was found but (g)(3) was not.
 listing.criteria.missingG3complement=(g)(3) was found without a required related certification.ired certification criteria 170.314 (g)(4) was not found.
+listing.criteria.missing170314g4=Required certification criteria 170.314 (g)(4) was not found.
+listing.criteria.missingUcdProccesses=Critiera % indicated SED but no UCD Processes were listed.
```

Comment:
Should this be `Criteria %s indicated...`, with the addition of `s`?

Reply:
The original error message did not have a plural option so I dont think so.